src/App.jsx

```diff
@@ -25,9 +25,10 @@ class App extends Component {
     return (
       <div className="App">
         <div className="container-fluid">
+          <div className="header">Header</div>

```

Comment:
What's the reasoning for the header?

Reply:
Not sure if header is outlined in Product overview. It seems out of scope of product overview so I figured that we still need a small space for it at the top (logo left & search right). Maybe I am wrong and it is in your row. Lmk

Reply:
Yeah I know what you're talking about now. There was anything in the product overview section about, but once I finish my shit I can take care of that